deployment/terraform/create.go

```diff
@@ -743,13 +744,14 @@ func (t *Terraform) setupProxyServer(extAgent *ssh.ExtAgent) {
 			{srcData: strings.TrimLeft(serverSysctlConfig, "\n"), dstPath: "/etc/sysctl.conf"},
 			{srcData: strings.TrimLeft(nginxConfig, "\n"), dstPath: "/etc/nginx/nginx.conf"},
 			{srcData: strings.TrimLeft(limitsConfig, "\n"), dstPath: "/etc/security/limits.conf"},
+			{srcData: strings.TrimPrefix(prometheusNodeExporterConfig, "\n"), dstPath: "/etc/default/prometheus-node-exporter"},
 		}
 		if err := uploadBatch(sshc, batch); err != nil {
 			mlog.Error("batch upload failed", mlog.Err(err))
 			return
 		}
 
-		cmd := "sudo sysctl -p && sudo service nginx restart"
+		cmd := "sudo ethtool -G $(ip route show to default | awk '{print $5}') rx 8192 && sudo sysctl -p && sudo service nginx restart"
```

Comment:
Adding a comment is the easiest. We can also document it separately, but then it's two places to keep up to date. And, of course, we may need to surface this in our documentation eventually if customers want to follow our recommendations.

Reply:
@agarciamontoro I added a comment for now and made it dynamic since the higher value (8192) may not work for all instance types. We may surface it in config later if we need an extra level of tuning.

Reply:
Neat, thanks!